<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ioLaunch</class>
 <widget class="QMainWindow" name="ioLaunch">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>4</horstretch>
    <verstretch>2</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>360</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>450</width>
    <height>360</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>ioLaunch 0.2</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs.qrc">
    <normaloff>:/imgs/iolICO.png</normaloff>:/imgs/iolICO.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <widget class="QPushButton" name="btnLaunch">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>279</y>
      <width>451</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Arial Black&quot;;
color: #7fa0d3;</string>
    </property>
    <property name="text">
     <string>Launch</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>471</width>
      <height>281</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>451</width>
      <height>0</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">#tabMain{
	background-image: url(:/imgs/iol.png);
	background-repeat: no-repeat;
	background-position:center center;
}
QTabBar::tab{
	width: 150px;
}
QTabWidget::tab-bar{
	alignment: center;
}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabMain">
     <property name="minimumSize">
      <size>
       <width>445</width>
       <height>0</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <attribute name="title">
      <string>Launch</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabSettings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>441</width>
        <height>31</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="screenModeLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Screen Mode:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbFull">
         <property name="text">
          <string>Full Screen</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rbWin">
         <property name="text">
          <string>Windowed</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="resolutionLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Resolution:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbResolution">
         <item>
          <property name="text">
           <string>Native</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Custom</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>320x240</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>400x300</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>512x384</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>640x480</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>800x600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>960x720</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1024x768</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1152x864</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1280x1024</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1600x1200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2048x1536</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>856x480(16:9)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1280x720(16:9)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1920x1080(16:9)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1280x800(16:10)</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>W</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbWidth">
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbHeight">
         <property name="maximum">
          <number>100000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>441</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="installWizardLayout">
       <item>
        <widget class="QPushButton" name="btnRunInstallWizard">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Run Install Wizard</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabRend2">
     <attribute name="title">
      <string>Rend2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
